cmp/grid/GridModel.js

```diff
@@ -400,8 +400,8 @@ export class GridModel {
     }
 
     /** Update the underlying store. */
```

Comment:
could consider whether to trampoline the additional new store methods in this way as well